docs/source/usage/index.rst

```diff
@@ -73,15 +73,209 @@ When opening AiiDAlab you see a list of installed applications.
+
+       ---
+
+       **Step 4: Start the app from the start page.**
+
+       The newly installed app should now show up on the start page.
+
+       .. image:: ../_static/app-management-start-page.png
+
+       Each app banner also shows an indicator about whether there is an update available (see screenshot above).
+       To *update the app*, click on **Manage App** and then on the **Update** buttons.
+
+
+.. tabbed:: Terminal
+
+    .. panels::
+       :container: container-lg pb-3
+       :column: col-lg-12 p-2
+
+       **Step 1: Open the Terminal.**
+
+       Open the :fa:`terminal` by clicking on the corresponding icon in the nav bar.
+
+        .. image:: ../_static/nav-bar-terminal.png
+
+       ---
+
+       **Step 2: Install the app with the aiidalab command**
+
+       .. code-block:: console
+
+          ~$ aiidalab install <app-name>
+
+       Replace ``<app-name>`` with the name of the app you would like to install, e.g., ``aiidalab install quantum-espresso``.
+       Use ``aiidalab search`` to search among available apps and their versions.
+       Similarly, the ``aiidalab list`` lists all currently installed apps and their versions.
+
+
+.. _app-store:upgrade:
+
+Upgrade (or downgrade) an app
+=============================
+
+.. tabbed:: App Store
+
+    .. panels::
+       :container: container-lg pb-3
+       :column: col-lg-12 p-2
+
+       **Step 1: Find the app you would like to upgrade on the start page.**
+
+       On the home app start page, simply look for the app you would like to upgrade.
+
+       .. image:: ../_static/app-management-start-page-upgrade-available.png
+
+       Click on the **Manage App** button to open the app manager.
+
+       ---
+
+       **Step 2: Open the App Management page**
+
+       The green :fa:`arrow-circle-up` **Update** button indicates that there is a newer version of the app available.
+
+       .. image:: ../_static/app-management-upgrade-available.png
+
+       Click on the :fa:`arrow-circle-up` **Update** button to upgrade the app.
+
+       By default, the app will be upgraded to the latest available version, howevever you can alternatively select any available version, including a version that is lower than the currently installed one.
+
+.. tabbed:: Terminal
+
+    Within the :fa:`terminal` Terminal, execute the following command to upgrade:
+
+    .. code-block:: console
+
+       ~$ aiidalab install <app-name>
+
+    This will install the most recent version of an app, regardless of whether it is already installed or not.
+    You will be prompted to confirm the operation.
+
+    You can install a specific version, by using standard `PEP 440 version specifiers`_, for example:
+
+    .. code-block:: console
+
+       ~$ aiidalab install quantum-espresso==v22.01.0
+
+.. _app-store:uninstall:
+
+Uninstall an app
+================
+
+Uninstalling an app works similar to upgrading or downgrading an app via the **Manage App** page or on the Terminal.
+
+.. tabbed:: App Store
```

Comment:
Although it's part of the App store, perhaps "App manager" is more clear for users here?

```suggestion
.. tabbed:: App manager
```